ESM build is incompatible with Node.js ESM

Hi,
the published ESM build of `@ucans/ucans` is wrong, it misses the `.js` file endings that are required for ESM imports.

To reproduce:
`package.json`
```json
{
  "type": "module",
  "dependencies": {
    "@ucans/ucans": "^0.11.0-alpha"
  }
}
```
`test.js`
```javascript
import * as ucans from '@ucans/ucans'
```
Run
```sh
$ node test.js
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/tmp/ucantest/node_modules/@ucans/default-plugins/dist/esm/prefixes' imported from /tmp/ucantest/node_modules/@ucans/default-plugins/dist/esm/rsa/crypto.js
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
}
```

--- a/packages/core/src/semver.ts
+++ b/packages/core/src/semver.ts
@@ -1,4 +1,4 @@
-import { hasProp, isRecord } from "./util"
+import { hasProp, isRecord } from "./util.js"
 
 
 // Types

--- a/packages/default-plugins/src/ed25519/crypto.ts
+++ b/packages/default-plugins/src/ed25519/crypto.ts
@@ -1,5 +1,5 @@
-import { EDWARDS_DID_PREFIX } from "../prefixes"
-import { didFromKeyBytes, keyBytesFromDid } from "../util"
+import { EDWARDS_DID_PREFIX } from "../prefixes.js"
+import { didFromKeyBytes, keyBytesFromDid } from "../util.js"
 
 export const didToPublicKey = (did: string): Uint8Array => {
   return keyBytesFromDid(did, EDWARDS_DID_PREFIX)

--- a/packages/default-plugins/src/rsa/crypto.ts
+++ b/packages/default-plugins/src/rsa/crypto.ts
@@ -1,6 +1,6 @@
 import { webcrypto } from "one-webcrypto"
 import * as uint8arrays from "uint8arrays"
-import { RSA_DID_PREFIX, RSA_DID_PREFIX_OLD } from "../prefixes"
+import { RSA_DID_PREFIX, RSA_DID_PREFIX_OLD } from "../prefixes.js"
 import { didFromKeyBytes, keyBytesFromDid } from "../util.js"
 
 export const RSA_ALG = "RSASSA-PKCS1-v1_5"

